src/t8_eclass.c

```diff
@@ -80,13 +80,13 @@ const int t8_edge_vertex_to_tree_vertex[T8_ECLASS_COUNT][T8_ECLASS_MAX_EDGES][2]
     { 0, 4 },
     { 1, 5 },
     { 2, 6 },
-    { 3, 7 } }, /* hex */
-  { { -1 } },   /* tet */
-  { { -1 } },   /* prism */
-  { { -1 } },   /* pyramid */
+    { 3, 7 } },                                                                                 /* hex */
+  { { -1 } },                                                                                   /* tet */
+  { { 1, 2 }, { 0, 2 }, { 0, 1 }, { 4, 5 }, { 3, 5 }, { 3, 4 }, { 1, 4 }, { 2, 5 }, { 0, 3 } }, /* prism */
```

Comment:
Where do these numbers come from?

Reply:
I came up with the edge numbering by my self. I had to do the same for tets, since there was no edge numbering before.
I kept the edge numbering for triangles (a triangle forms the base of the prism with the vertices 0, 1 and 2) for both triangles of the prism and then added the other 3 edges.

Reply:
The edge numbering of the base edges is not consistent with the edge numbering of the triangles. Furthermore, for both triangles and prisms, the edges switch the direction and do not form a circle. I think the should be oriented counterclockwise.

Reply:
As disscused this will be adressed in a new PR and is documented in issue #975